packages/vue-instantsearch/src/components/SearchInput.vue

```diff
@@ -66,15 +62,13 @@
       <slot name="reset-icon">
         <svg
           aria-hidden="true"
-          role="img"
-          xmlns="http://www.w3.org/2000/svg"
-          style="width: 1em; height: 1em"
+          height="10"
           viewBox="0 0 20 20"
           :class="suit('resetIcon')"
+          width="10"
```

Comment:
Does it still render correctly without the xmlns attribute?

Reply:
It's required only in `.svg` files but optional when embedded in HTML, and it already worked in React and vanilla so that's why I removed it